certified-connectors/FactSet/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1410 @@
+                        "description": "Associated Orgs"
+                      }
+                    }
+                  },
+                  "description": "Entities"
+                }
+              }
+            }
+          }
+        },
+        "summary": "NER - Entities",
+        "operationId": "ner-entities",
+        "parameters": [
+          {
+            "name": "body",
+            "in": "body",
+            "required": false,
+            "schema": {
+              "type": "object",
+              "properties": {
+                "text": {
+                  "type": "string",
+                  "description": "English plain text to extract named entities from. Maximum of 10,000 characters",
+                  "title": "Text"
+                },
+                "filterEntities": {
+                  "type": "boolean",
+                  "description": "Drop lower probability entities using machine learning filtering rules.",
+                  "title": "Filter Entities",
+                  "enum": [
+                    true,
+                    false
+                  ],
+                  "default": true
+                },
+                "enableIdLookup": {
+                  "type": "boolean",
+                  "description": "Drop lower probability entities using machine learning filtering rules.",
+                  "title": "Enable ID Lookup",
+                  "enum": [
+                    true,
+                    false
+                  ],
+                  "default": true
+                }
+              },
+              "required": [
+                "text"
+              ]
+            }
+          }
+        ],
+        "description": "Extract named entities from document text."
+      }
+    },
+    "/book-builder-api/v1/book-list": {
+      "get": {
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "type": "array",
+              "items": {
+                "type": "object",
+                "properties": {
+                  "bookID": {
+                    "type": "integer",
+                    "format": "int32",
+                    "description": "Book ID"
+                  },
+                  "book_name": {
+                    "type": "string",
+                    "description": "Book Name"
+                  },
+                  "created_on": {
+                    "type": "string",
+                    "description": "Created Date"
+                  }
+                }
+              }
+            }
+          }
+        },
+        "summary": "BookBuilder - Get Book List",
+        "description": "Retrieves the list of books that were previously created and are available in the client's book library",
+        "operationId": "get-book-list",
+        "parameters": []
+      }
+    },
+    "/book-builder-api/v1/template-list": {
+      "get": {
+        "responses": {
+          "200": {

```

Comment:
@krithsai please list all known 4xx/5xx responses for your operations if possible. You can refer to the documentation for [an example](https://docs.microsoft.com/en-us/connectors/custom-connectors/coding-standards#actions-and-triggers).